persistence/javadsl/src/main/java/com/lightbend/lagom/javadsl/AkkaTaggerAdapter.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.javadsl.persistence;
+
+import akka.cluster.sharding.typed.javadsl.EntityContext;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.function.Function;
+
+public class AkkaTaggerAdapter {
+
+  /**
+   * Adapts an existing Lagom {@code AggregateEventTagger} to a function {@code Function<Event,
+   * Set<String>>} as expected by Akka Persistence Typed {@code EventSourcedBehavior.withTagger}
+   * API.
+   */
+  public static <Event extends AggregateEvent<Event>> Function<Event, Set<String>> fromLagom(
+      EntityContext<?> entityContext, AggregateEventTagger<Event> lagomTagger) {
```

Comment:
Note that I'm passing the `EntityContext`. In order to make sure we will tag correctly, we should ask for the context and use the `entityId`. If it was a String, user could pass anything or even `PersistenceId.toString` which would be wrong.